src/resources/js/tickets-attendees.js

```diff
@@ -1,287 +1,290 @@
-jQuery( document ).ready( function( $ ) {
-
-	if ( typeof AttendeesPointer !== 'undefined' && null !== AttendeesPointer ) {
-		options = $.extend( AttendeesPointer.options, {
-			close: function() {
-				$.post( ajaxurl, {
-					pointer: AttendeesPointer.pointer_id,
-					action : 'dismiss-wp-pointer'
-				} );
-			},
-			open: function( event, widget ) {
-				widget.pointer
-					.css({
-						top: parseInt( widget.pointer.css( 'top' ).replace( 'px', '' ), 10 ) + 5
-					})
-					.find( '.wp-pointer-arrow' ).css({
+var tribe_event_tickets_attendees = tribe_event_tickets_attendees || {};
+
+( function( $, obj ) {
```

Comment:
Yeah I veered away from `my` because it can mean local in some other languages. I also wondered about using `public` but it sounds like it could become an actual keyword in future so might not be the brightest idea.